release-team/README.md

```diff
@@ -79,7 +79,7 @@ there are 3 types of Kubernetes releases:
 | Test Infra | [Test Infra Handbook](role-handbooks/test-infra/README.md) |
 | Bug Triage | [Bug Triage Handbook](role-handbooks/bug-triage/README.md) |
 | Branch Manager | [Branch Manager Handbook](role-handbooks/branch-manager/README.md) |
-| Docs | [Doc Handbook](role-handbooks/docs/README.md) |
+| Docs | [Doc Handbook](role-handbooks/documentation-guides) |
```

Comment:
Please remove this change.
We should fix the broken link by conforming the Docs handbook to the other role handbooks.